matlab/schemas/+experiment/ProjectorDisplay.m

```diff
@@ -0,0 +1,12 @@
+%{
+experiment.ProjectorDisplay (Manual) # 
```

Comment:
same here for the first line not being very descriptive

Reply:
same as above